enabler/src/components/payment-methods/easycredit/easycredit.ts

```diff
@@ -0,0 +1,72 @@
+import { BaseOptions } from '../../../payment-enabler/payment-enabler-mock';
+import {
+  ComponentOptions,
+  PaymentComponent,
+  PaymentComponentBuilder,
+  PaymentMethod
+} from '../../../payment-enabler/payment-enabler';
+import { BaseComponent } from "../../base";
+import { importEasyCreditScript } from '../../../utils/app.utils';
+
+export class EasyCreditBuilder implements PaymentComponentBuilder {
+  public componentHasSubmit = true
+
+  constructor(private baseOptions: BaseOptions) {}
+
+  build(config: ComponentOptions): PaymentComponent {
+    return new EasyCredit(this.baseOptions, config);
+  }
+}
+
+export class EasyCredit extends BaseComponent {
+  private baseOptions: BaseOptions
+  
+  constructor(baseOptions: BaseOptions, componentOptions: ComponentOptions) {
+    super(PaymentMethod.easycredit, baseOptions, componentOptions);
+    this.baseOptions = baseOptions;
+  }
+
+
+  async mount(selector: string) {
+    importEasyCreditScript();
+
+    const response = await this.getPaymentMethod();
+
+    if (response) {  
+      document.querySelector(selector).insertAdjacentHTML("afterbegin", this._getTemplate(response));
```

Comment:
what if this selector not existing on the DOM?

Reply:
I updated it, we would throw an error, then log it to the console tab if it happens, then there will be nothing rendered